ground/uavobjgenerator/uavobjectparser.cpp

```diff
@@ -404,6 +404,12 @@ QString UAVObjectParser::parseXML(QString& xml, QString& filename)
         // Calculate size
         calculateSize(info);
 
+        // Buffer on flight side is 256 bytes
+        // Additional overhead is 2+4+2+2+1+1=12

```

Comment:
It would be nice to have the overhead bytes explicitly defined here in the comment.